src/components/QueryableLeaderboard.tsx

```diff
@@ -0,0 +1,28 @@
+"use client"
+
+// import { ProjectsTable } from "@/components/ProjectsTable";
+import hero from "../images/herotitle.png";
+import { Tab, Tabs, TabList, TabPanel } from "react-tabs";
+import { useState } from "react";
+import Details from "./Details";
+import { ProjectWithRank } from "../services/ezrfApi/types";
+import { ProjectsTable } from "./ProjectsTable/ProjectsTable";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { Leaderboard } from "./Leaderboard";
+import { ProjectsProvider } from "../contexts/projectsContext";
+
+
+export function QueryableLeaderboard() {
+
+
+    const queryClient = new QueryClient();
```

Comment:
It's better to have the provider in `app/layout.tsx` so it's used across all pages.

https://tanstack.com/query/latest/docs/framework/react/guides/advanced-ssr#server-components-and-nextjs-app-router

Reply:
Awesome - yeah I was rushing and wanted to get it running locally, I will clean this up!